car-service/src/main/kotlin/com/curious/programmers/carservice/CarService.kt

```diff
@@ -0,0 +1,12 @@
+package com.curious.programmers.carservice
+
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+
+
+interface CarService {
```

Comment:
As almost always, it'll be a philo question, but do we really need interfaces for services in this particular case?